integration/kubernetes/k8s-limit-range.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	get_pod_config_dir
+	namespace_name="default-cpu-example"
+	pod_name="default-cpu-test"
+}
+
+@test "Limit range for storage" {
+	# Create namespace
+	kubectl create namespace "$namespace_name"
+
+	# Create the LimitRange in the namespace
+	kubectl create -f "${pod_config_dir}/limit-range.yaml" --namespace=${namespace_name}
+
+	# Create the pod
+	kubectl create -f "${pod_config_dir}/pod-cpu-defaults.yaml" --namespace=${namespace_name}
+
+	# Get pod specification
+	kubectl wait --for=condition=Ready pod "$pod_name" --namespace="$namespace_name"
+
+	# Check limits
+	kubectl describe pod "$pod_name" --namespace="$namespace_name" | grep "500m"
```

Comment:
Is this 500Mb or minutes or microseconds or miles or metres or mangos? :) I'm assuming this magic number is somehow related to the static yaml.

I do wonder if we should be generating that YAML from `.yaml.in` files for all these k8s tests so that the YAML is guaranteed to match the test script but I guess we can save that for another day.

Reply:
@jodh-intel that is 500 millicpus and yes it is related to the static.yaml

Reply:
I think @jodh-intel is really asking you to make it clear in the code - so, an explanatory comment for the 'magic numbers' :-)

Reply:
@grahamwhaley and @jodh-intel changes applied

Reply:
Thanks @GabyCT - that's much clearer. Of course, it would be great if `kubectl` was a little saner, but that's another story... :-)